tests/test_MocLogin.py

```diff
@@ -0,0 +1,101 @@
+from django.test import TestCase
+from core.models import AtmosphereUser as User
+from core.models import Provider
+from core.models import PlatformType
+from core.models import ProviderType
+
+from threepio import logger
+
+from atmosphere import settings
+
+
+# This is assuming a clean database
+class MocLogin(TestCase):
+    username = test_settings['username']
```

Comment:
You are testing to login to GIJI and you are not testing to login to OpenStack via keystone since that park should be taken care by a "try-except" block. We are not writing unit test scripts for keystone since that part has been taken care by upstream. We are writing unit test scripts for GIJI. And to login GIJI currently, you need to have `user`, `identity`, `provider`, `credential` and that's why you are feeding values to them also.